indico/util/string.py

```diff
@@ -27,6 +27,7 @@
 import html5lib
 import markdown
 import translitcodec
+import unidecode
```

Comment:
I'd prefer `from unicode import unicode` for this one. In case of translitcodec we don't do that because `long_encode` is not particularly meaningful as an unqualified name.